esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -727,6 +727,37 @@ void EspHttpBinding::addToESPCache(CHttpRequest* request, CHttpResponse* respons
         ESPLOG(LogMax, "Failed to add ESP Cache for %s.", method);
 }
 
+void EspHttpBinding::clearCacheByGroupID(const char *ids)
+{
+    if (isEmptyString(ids))
+        return;
+
+    IEspContainer *espContainer = nullptr;
+    StringArray idList;
+    idList.appendListUniq(ids, ",");
+    ForEachItemIn(i, idList)
+    {
+        const char *id = idList.item(i);
+        if (espCacheClient && strieq(id, getCacheGroupID()))
+        {
+            espCacheClient->flush(0);
+            continue;
+        }
+
+        Owned<IEspCache> cacheClient;
+        if (!espContainer)
+            espContainer = getESPContainer();
+        const StringAttr *initString = espContainer->queryCacheInitString(id);
+        if (initString && !initString->isEmpty())
+        {
+            cacheClient.setown(createESPCache(initString->str()));
```

Comment:
Is this operation expensive?  From the code it looks like something that should only be done infrequently.

Reply:
I may move the creations to the beginning when ESP starts.